colony/commands/sb.py

```diff
@@ -151,63 +190,70 @@ def do_start(self):
 
         except Exception as e:
             logger.debug(f"Unable to obtain default values. Details: {e}")
+        return repo
+
+    def generate_sandbox_name(self, blueprint_name: str, temp_working_branch: str, working_branch: str) -> str:
```

Comment:
why does it needs to be in this file and as a public method?
its better to move this logic to a different file